CoffeeChat/Controller/EditDemographicsViewController.swift

```diff
@@ -96,7 +117,7 @@ class EditDemographicsViewController: UIViewController {
         nameTextField.backgroundColor = .backgroundWhite
         nameTextField.textColor = .black
         nameTextField.font = ._20CircularStdBook
-        nameTextField.text = name
+        nameTextField.text = "\(user.firstName) \(user.lastName)"
         nameTextField.clearButtonMode = .never
         nameTextField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 12, height: 49))
         nameTextField.leftViewMode = .always
```

Comment:
since the name textfield has a leftView padding maybe add a rightView padding too ?